### ISSUE TITLE
Review and Give Feedback of Gecko Analysis Notebook ✨

### ISSUE THREAD

**Issue by jonathanferrari** [ds-modules/IB-C32#7]
# Gecko Notebook Review 🦎📓📋 

## TL;DR
- review #6  and #2 then open the notebook via the link in the [README](https://github.com/ds-modules/IB-C32/blob/gecko/gecko/README.md)

- Leave detailed feedback on the pull request (#6) and a summary of your feedback on this thread

## Assignees:
@JessicaGolden @zcjanin @tanyagrawal

## Deadline:
<h3>Wednesday, February 15<sup>th</sup>, 2023</h3>

## Action Items:
- ### Review:
    - The open issue for the creation of the introductory notebook: #2 
    - The pull request to merge the `gecko` branch into `main`: #6
    - Open the notebook in DataHub via the link in the [README](https://github.com/ds-modules/IB-C32/blob/gecko/gecko/README.md)
- ### Critique:
    - Please leave constructive criticism on the pull request as a comment
    - If there is something that you think should be fixed, how would you fix it?
    - The students who will be using the notebook have little to no coding experience
    - Overall impression of the notebook through the lens of the students who will use the notebook
- Leave a summary of your critique as a comment on this issue

## Details: 
Please be as detailed as possible! Any and all feedback is welcome, especially things that this notebook can do better or weaknesses that you see.

Examples of what feedback you can leave might be:
- typos that you found
- something was hard to understand (or would be for the students)
- was any of the formatting hard to follow
- was the notebook too long
- should I have gone more in-depth into anything
- is there anything I spent too much time explaining
- any other improvements you may have noticed

Of course, positive comments are more than welcome, but don't be afraid to give constructive feedback!

## Conclusion:
Thank you all, and if you have any questions/concerns/comments please leave them here, or in our slack channel; I will actively review both!

**Comment by JessicaGolden** [comment]
Feedback summary:

It was an informative notebook and I had a lot of fun reading through it. I just spotted a few typos and discovered a way to make the plot more visible for students so they can more easily screenshot the plot if needed.

**Comment by tanyagrawal** [comment]
Overall, the notebook was very organized and had a lot of step-by-step explanations for conceptual concepts as well as the graphing portion. I thought it had the right amount of detail and instructions for the students to follow and as Jessica mentioned, it was very informative. I don't have any feedback in terms of improvement and it looks good!

**Comment by zcjanin** [comment]
Feedback:

I think the overall structure and flow of the notebook is very logical and coherent for a student with little/no coding experience. I think it is very user-friendly and provides a good introduction on the experiment. I also think that the way the data is presented does not come off as intimidating for students. I enjoyed going through this notebook!
